### PR TITLE
Develop

### PR DESCRIPTION
**Pull request by ptichkaa**

### CODE REVIEW

File: app/src/main/res/drawable/round_background.xml [hunk 1]
```diff
@@ -1,6 +1,6 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <shape xmlns:android="http://schemas.android.com/apk/res/android">
 <corners
-    android:radius="15dp" />
+    android:radius="16dp" />
```
Comment (CodOndinnonk) [inline app/src/main/res/drawable/round_background.xml:4]: dimen

File: app/src/main/res/layout/activity_main.xml [hunk 2]
```diff
@@ -21,22 +21,21 @@
         android:id="@+id/main_title"
         android:layout_width="wrap_content"
         android:layout_height="wrap_content"
-        android:text="Samsung Health"
+        android:text="@string/main_title"
         android:textColor="@color/black"
-        android:textSize="37sp"
+        android:textSize="@dimen/text_big"
+        app:layout_constraintBottom_toTopOf="@+id/steps_container"
         app:layout_constraintLeft_toLeftOf="parent"
```
Comment (CodOndinnonk) [inline app/src/main/res/layout/activity_main.xml]: start to start

File: app/src/main/res/layout/activity_main.xml [hunk 3]
```diff
@@ -21,22 +21,21 @@
...
-        app:layout_constraintTop_toTopOf="parent"
-        app:layout_constraintBottom_toTopOf="@+id/steps_container"
-        />
+        app:layout_constraintTop_toTopOf="parent" />
 
     <TextView
         android:id="@+id/hint"
         android:layout_width="wrap_content"
         android:layout_height="wrap_content"
         android:text="Синхронизировано с Gear S3 в 01:53"
```
Comment (CodOndinnonk) [inline app/src/main/res/layout/activity_main.xml:36]: string

File: app/src/main/res/layout/activity_main.xml [hunk 4]
```diff
@@ -59,7 +58,6 @@
         android:background="@drawable/round_background"
         android:padding="15dp"
```
Comment (CodOndinnonk) [inline app/src/main/res/layout/activity_main.xml]: dimen

File: app/src/main/res/layout/activity_main.xml [hunk 5]
```diff
@@ -104,28 +102,30 @@
             android:layout_height="wrap_content"
             android:text="Все шаги, 02:14"
             android:textColor="#9C9C9C"
```
Comment (CodOndinnonk) [inline app/src/main/res/layout/activity_main.xml:104]: +

File: app/src/main/res/values-ru/strings.xml [hunk 6]
```diff
@@ -0,0 +1,15 @@
...
+
+    <string name="main_title">Здоровье с Samsung</string>
+    <string name="hint">Синхронизировано с Gear S3 в 01:53</string>
+    <string name="steps_title">Шаги</string>
+    <string name="steps_time_total">Все шаги, 02:14</string>
+    <string name="practice_title">Упражнение</string>
+    <string name="practice_hint">Просмотреть журнал</string>
+    <string name="sleep_title">Вы спали?</string>
+    <string name="sleep_time">8 ч. 30 мин. |  02:10 - 10:40</string>
+    <string name="sleep_ok_btn">ОК</string>
```
Comment (CodOndinnonk) [inline app/src/main/res/values-ru/strings.xml]: delete

File: app/src/main/res/values/colors.xml [hunk 7]
```diff
@@ -7,4 +7,9 @@
     <color name="teal_700">#FF018786</color>
     <color name="black">#FF000000</color>
     <color name="white">#FFFFFFFF</color>
+    <color name="background_color">#F5F5F5</color>
+    <color name="grey_text_color">#9C9C9C</color>
+    <color name="progress_bar">#21A667</color>
```
Comment (CodOndinnonk) [inline app/src/main/res/values/colors.xml]: fix

File: app/src/main/res/values/dimens.xml [hunk 8]
```diff
@@ -0,0 +1,18 @@
+<?xml version="1.0" encoding="utf-8"?>
+<resources>
+
+    <dimen name="text_big">34sp</dimen>
+    <dimen name="text_medium">18sp</dimen>
+    <dimen name="text_small">16sp</dimen>
+    <dimen name="text_extra_small">12sp</dimen>
+
+    <dimen name="padding_constraint_layout">10dp</dimen>
```
Comment (CodOndinnonk) [inline app/src/main/res/values/dimens.xml]: fix name

File: app/src/main/res/values/dimens.xml [hunk 9]
```diff
@@ -0,0 +1,18 @@
...
+    <dimen name="white_container_padding">20dp</dimen>
```
Comment (CodOndinnonk) [inline app/src/main/res/values/dimens.xml:9]: fix

File: app/src/main/res/values/dimens.xml [hunk 10]
```diff
@@ -0,0 +1,18 @@
...
+    <dimen name="margin_settings">10dp</dimen>
```
Comment (CodOndinnonk) [inline app/src/main/res/values/dimens.xml]: need check

File: app/src/main/res/values/strings.xml [hunk 11]
```diff
@@ -1,3 +1,13 @@
...
     <string name="app_name">My Application</string>
+    <string name="main_title">Samsung Health</string>
+    <string name="hint">Synchronized with Gear S3 at 01:53</string>
+    <string name="steps_title">Steps</string>
+    <string name="steps_time_total">All steps, 02:14</string>
+    <string name="practice_title">An exercise</string>
+    <string name="practice_hint">View log</string>
+    <string name="sleep_title">You slept?</string>
+    <string name="sleep_time">8 hours 30 minutes | 02:10 - 10:40</string>
+    <string name="sleep_ok_btn">OK</string>
```
Comment (CodOndinnonk) [inline app/src/main/res/values/strings.xml]: ok

File: app/src/main/res/layout/activity_main.xml (hunk 2 above)
Comment (ptichkaa) [reply app/src/main/res/layout/activity_main.xml]: +

File: app/src/main/res/layout/activity_main.xml (hunk 4 above)
Comment (ptichkaa) [reply app/src/main/res/layout/activity_main.xml]: +

File: app/src/main/res/values-ru/strings.xml (hunk 6 above)
Comment (ptichkaa) [reply app/src/main/res/values-ru/strings.xml]: +

File: app/src/main/res/values/colors.xml (hunk 7 above)
Comment (ptichkaa) [reply app/src/main/res/values/colors.xml]: +

File: app/src/main/res/values/dimens.xml (hunk 8 above)
Comment (ptichkaa) [reply app/src/main/res/values/dimens.xml]: +

File: app/src/main/res/values/dimens.xml (hunk 9 above)
Comment (ptichkaa) [reply app/src/main/res/values/dimens.xml:9]: +

File: app/src/main/res/values/dimens.xml (hunk 10 above)
Comment (ptichkaa) [reply app/src/main/res/values/dimens.xml]: +

File: app/src/main/res/values/strings.xml (hunk 11 above)
Comment (ptichkaa) [reply app/src/main/res/values/strings.xml]: +

File: app/src/main/res/values/dimens.xml [hunk 12]
```diff
@@ -0,0 +1,17 @@
+<?xml version="1.0" encoding="utf-8"?>
+<resources>
+
+    <dimen name="text_big">34sp</dimen>
+    <dimen name="text_medium">18sp</dimen>
+    <dimen name="text_small">16sp</dimen>
+    <dimen name="text_extra_small">12sp</dimen>
+    <dimen name="padding_ALLconstraint_layout">10dp</dimen>
```
Comment (CodOndinnonk) [inline app/src/main/res/values/dimens.xml]: wrong key style

File: app/src/main/res/layout/activity_main.xml [hunk 13]
```diff
@@ -3,129 +3,129 @@
...
             app:layout_constraintTop_toBottomOf="@+id/steps_title"
             app:layout_constraintVertical_bias="0.75" />
 
         <TextView
             android:id="@+id/steps_time_total"
             android:layout_width="wrap_content"
             android:layout_height="wrap_content"
-            android:text="Все шаги, 02:14"
+            android:text="@string/steps_time_total"
             android:textColor="#9C9C9C"
```
Comment (CodOndinnonk) [inline app/src/main/res/layout/activity_main.xml]: +

File: app/src/main/res/values/dimens.xml (hunk 12 above)
Comment (ptichkaa) [reply app/src/main/res/values/dimens.xml]: +

File: app/src/main/res/layout/activity_main.xml (hunk 13 above)
Comment (ptichkaa) [reply app/src/main/res/layout/activity_main.xml]: +

**Pull request merged**